.buildkite/pipeline.perf-tests.yml

```diff
@@ -0,0 +1,30 @@
+# yaml-language-server: $schema=https://raw.githubusercontent.com/buildkite/pipeline-schema/main/schema.json
+
+env:
+  DOCKER_REGISTRY: "docker.elastic.co"
+  DOCKER_IMAGE: "${DOCKER_REGISTRY}/observability-ci/fleet-server" # needs to rename for rollback
+  DOCKER_IMAGE_GIT_TAG: "${BUILDKITE_BRANCH}" # needs to rename for rollback
+  DOCKER_IMAGE_LATEST_TAG: "latest" # needs to rename for rollback
+  DOCKER_IMAGE_SHA_TAG: "git-${BUILDKITE_COMMIT:0:12}" # needs to rename for rollback, should be "git-${BUILDKITE_COMMIT:0:12}"
+
+steps:
+  - label: ":docker: Publish docker image"
+    key: "create-image"
+    command: ".buildkite/scripts/build_push_docker_image.sh"
```

Comment:
As it is using `observability-ci` namespace, it is needed to add the required credentials in `.buildkite/hooks/pre-command`.

Something like this but using the keys for your new pipeline slug (`fleet-server-perf-tests`) and step keys (`create-image`):
https://github.com/elastic/fleet-server/blob/bf17db29514f0d90e22587929dc1469f830dd278/.buildkite/hooks/pre-command#L43-L49

And the same for the `pre-exit` hook:
https://github.com/elastic/fleet-server/blob/bf17db29514f0d90e22587929dc1469f830dd278/.buildkite/hooks/pre-exit#L7-L11